po/de.po

```diff
@@ -185,24 +186,24 @@ msgid ""
 "This window has several terminals open. Closing the window will also close "
 "all terminals within it."
 msgstr ""
-"In diesem Fenster sind mehrere Terminal geöffnet. Beim schließen des "
-"Fensters werden auch die Terminal darin geschlossen."
+"In diesem Fenster sind mehrere Terminals geöffnet. Das Schließen dieses "
+"Fensters wird auch die Terminals darin schließen."
 
 #: ../terminatorlib/container.py:178
 msgid ""
 "This tab has several terminals open. Closing the tab will also close all "
 "terminals within it."
 msgstr ""
-"In diesem Tab sind mehrere Terminal geöffnet. Beim schließen des Tab werden "
-"auch die Terminal darin geschlossen."
+"In diesem Reiter sind mehrere Terminals geöffnet. Das Schließen dieses "
+"Reiters wird auch die Terminals darin schließen."
 
 #: ../terminatorlib/container.py:198
 msgid "Do not show this message next time"
-msgstr "Diese Nachricht das nächste Mal nicht anzeigen"
+msgstr "Diese Nachricht in Zukunft nicht mehr anzeigen"
 
 #: ../terminatorlib/encoding.py:34
 msgid "Current Locale"
-msgstr "Derzeitige Standorteinstellungen"
+msgstr "Derzeitiger Standort"
```

Comment:
Probably it's more a "Derzeitige Spracheinstellungen"?

Reply:
I am not sure about the correct translation of "Locale"

Reply:
Do you know about the context?

Reply:
Usually the context for the use of the term "locale" is about language or charset encoding. That's why I was suggesting "Spracheinstellungen". An alternative which may be even better would be "(momentaner) Zeichensatz" or something similar.

A quick look into the code told me that this translation is about the context menu when you right click somewhere in the terminal, there is a menu point "Encodings" which is most properly translated as "Zeichensatz" in my opinion.

Reply:
so I found https://de.wikipedia.org/wiki/Locale where Locale is the umbrella term for all these settings location language, keyboard layout etc.. They translate it with "Einstellungssatz".

Reply:
"Einstellungssatz"… yeah, I like that word, even if it sounds a bit unusual to me personally. But seems it's the most proper translation then. (:

Reply:
Okay I will use that :)